README.md

```diff
@@ -321,17 +321,22 @@ find ~/.local/share/pikaur/aur_repos -mindepth 1 -maxdepth 1 -type d | xargs -r
 
 ##### How to downgrade a package?
 
-This will show a list of commits to choose one to downgarade to.
+Run the following to:
+1) Clone the PKGBUILD repo and show a log of commits to choose from
+2) Choose a commit hash to downgrade to
+3) Build and install the downgraded PKGBUILD
+4) Use `makepkg` if `pikaur -P` fails to install
```

Comment:
`pikaur -P` will always "fail" to install 😺 - you need to use `-i` flag for installing